ConsoleGameTests.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.Test;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+public class ConsoleGameTests {
```

Comment:
I think there should be more tests here as you are doing more than displaying the board..